File: Python/Seniman.py
class Seniman:
    def __init__(self, nama="", umur="", sejak=""):
        self.nama_seniman = nama
        self.umur_seniman = umur
        self.sejak = sejak

    def set_nama(self, nama):
        self.nama_seniman = nama

    def get_nama(self):
        return self.nama_seniman

    def set_umur(self, umur):
        self.umur_seniman = umur

    def get_umur(self):
        return self.umur_seniman

    def set_sejak(self, sejak):
        self.sejak = sejak

    def get_sejak(self):
        return self.sejak

    def add_data_seniman(self, nama, umur, sejak):
        self.nama_seniman = nama
        self.umur_seniman = umur
        self.sejak = sejak

File: javanies/Seniman.java
public class Seniman {
    protected String namaSeniman;
    protected String umurSeniman;
    protected String sejak;

    public Seniman() {}

    public Seniman(String nama, String umur, String sejak) {
        this.namaSeniman = nama;
        this.umurSeniman = umur;
        this.sejak = sejak;
    }

    public void setNama(String nama) { this.namaSeniman = nama; }
    public String getNama() { return namaSeniman; }

    public void setUmur(String umur) { this.umurSeniman = umur; }
    public String getUmur() { return umurSeniman; }

    public void setSejak(String sejak) { this.sejak = sejak; }
    public String getSejak() { return sejak; }

    public void addDataSeniman(String nama, String umur, String sejak) {
        this.namaSeniman = nama;
        this.umurSeniman = umur;
        this.sejak = sejak;
    }
}
